packages/fiber/src/core/hooks.tsx

```diff
@@ -103,7 +110,7 @@ function loadingFn<L extends LoaderProto<any>>(
             ),
           ),
       ),
-    )
+    ).finally(() => (loader as any).dispose?.())
```

Comment:
hey @CodyJasonBennett this breaks using `DRACOLoader` directly (`useLoader(DRACOLoader, url)`, without `useGTLF()` or similar), since `DRACOLoader` cannot be used again after dispose https://threejs.org/docs/#examples/en/loaders/DRACOLoader.dispose

At the moment I simply workaround this by making `dispose()` not do anything.

While debugging this issue I also checked #3131, and I think it won't work as expected, I think `dispose()` should not be called if the user wants to reuse a particular loader.

What do you think?

Reply:
Yes, I agree. This was an oversight.

Reply:
Removed in https://github.com/pmndrs/react-three-fiber/releases/tag/v8.16.2.